Makefile

```diff
@@ -33,62 +30,127 @@ else
 	mkdir -p ${BUILD_DIR}
 endif
 
-# Simple things, pointers into our build
-launcher: .pre-build
-	go run cmd/make/make.go -targets=launcher -linkstamp
+##
+## Build
+##
 
-table.ext: .pre-build
-	go run cmd/make/make.go -targets=table-extension -linkstamp
-table.ext-windows: .pre-build deps
-	go run cmd/make/make.go -targets=table-extension -linkstamp --os windows
+build_%: TARGET =  $(word 2, $(subst _, ,$@))
```

Comment:
for future reference for myself (and any other make novices): https://www.gnu.org/software/make/manual/html_node/Text-Functions.html

Reply:
Yep. I do that all over our Makefiles. I think it's more clever than I like, but make.